src/components/expression/expression.js

```diff
@@ -172,6 +204,22 @@ genotet.ExpressionView = function(viewName, params) {
     .on('genotet.updatePanel', function() {
       this.panel.dataLoaded();
     }.bind(this));
+
+  // Set up link callbacks.
+  $(this).on('genotet.addProfile', function(event, data) {
+    var genes = data.id.split(',');
```

Comment:
It is... It is not me that set this attribute. However, I converted it to an array before signaling.